proto/canal/EntryProtocol.pb.go

```diff
@@ -5,10 +5,11 @@ package com_alibaba_otter_canal_protocol
 
 import (
 	fmt "fmt"
-	proto "github.com/golang/protobuf/proto"
 	io "io"
 	math "math"
 	math_bits "math/bits"
+
+	proto "github.com/golang/protobuf/proto"
```

Comment:
recover this file?

Reply:
sorry ,I don't know what's you mean,this pr has nothing to do with proto package.

Reply:
I mean, undo the changes to this document.